Sources/NIO/FileHandle.swift

```diff
@@ -81,12 +81,56 @@ public final class NIOFileHandle: FileDescriptor {
 }
 
 extension NIOFileHandle {
+    /// `Mode` represents file access modes.
+    public struct Mode: OptionSet {
+        public let rawValue: UInt8
```

Comment:
This should probably be `private var` instead of `public let`.

Reply:
This is a requirement from `RawRepresentable`, we cannot make it private